crates/bevy_window/Cargo.toml

```diff
@@ -22,12 +22,7 @@ bevy_reflect = [
 ]
 
 ## Adds serialization support through `serde`.
-serialize = [
-  "serde",
-  "smol_str/serde",
-  "bevy_ecs/serialize",
-  "bevy_input/serialize",
-]
+serialize = ["serde", "bevy_ecs/serialize", "bevy_input/serialize"]
```

Comment:
i question the utility of these serialize features which just enable stuff in transitive deps. do we need these? they don't even do anything to this crate itself

Reply:
ive left them for now but we should probably reconsider them